lib/Runtime/Library/ModuleRoot.cpp

```diff
@@ -406,29 +406,29 @@ namespace Js
         return this->GetLibrary()->GetGlobalObject()->GlobalObject::DeleteRootProperty(propertyId, flags);
     }
 
-    BOOL ModuleRoot::HasItem(uint32 index)
+    PropertyQueryFlags ModuleRoot::HasItemQuery(uint32 index)
     {
-        return DynamicObject::HasItem(index)
-            || (this->hostObject && JavascriptOperators::HasItem(this->hostObject, index));
+        return JavascriptConversion::BooleanToPropertyQueryFlags(JavascriptConversion::PropertyQueryFlagsToBoolean((DynamicObject::HasItemQuery(index)))
```

Comment:
>JavascriptConversion::PropertyQueryFlagsToBoolean [](start = 65, length = 49)

Seems like an unnecessary conversion here, maybe BooleanToPropertyQueryFlags can just handle this natively since Property_NotFound == FALSE == 0 and everything else is TRUE